templates/download/desktop/index.html

```diff
@@ -50,7 +50,7 @@ <h2>Ubuntu {{ releases.latest.full_version }}</h2>
       <div class="col-8 p-divider__block">
         <p class="u-no-padding--top">The latest version of the Ubuntu operating system for desktop PCs and laptops, Ubuntu {{ releases.latest.full_version }} comes with nine months, until {{ releases.latest.eol }}, of security and maintenance updates.</p>
         <p>Recommended system requirements are the same as for Ubuntu {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr>.</p>
-        <p><a href="https://wiki.ubuntu.com/{{ releases.latest.slug }}/ReleaseNotes" class="p-link--external">Ubuntu {{ releases.latest.short_version }} release notes</a></p>
+        <p><a href="{{ releases.latest.release_notes_url }}" class="p-link--external">Ubuntu {{ releases.latest.short_version }} release notes</a></p>
```

Comment:
Does this need external if linking to the local discourse?

Reply:
probably not, will update